test/api.js

```diff
@@ -3,34 +3,36 @@ import assert from 'node:assert'
 import { getDefaultRootDirs } from '../lib/paths.js'
 
 describe('API', () => {
-  describe('Core()', () => {
+  describe('Core', () => {
     it('can be imported', () => {
       assert(Core)
     })
-    it('can be constructed', () => {
-      assert(new Core())
+  })
+  describe('Core.create', () => {
+    it('can be constructed', async () => {
+      assert(await Core.create())
     })
-    it('can be constructed with custom paths', () => {
-      assert(new Core(getDefaultRootDirs()))
+    it('can be constructed with custom paths', async () => {
+      assert(await Core.create(getDefaultRootDirs()))
     })
   })
```

Comment:
Can you please add a test to verify that `Core.create` creates the necessary directories and files?

Ideally, such test should call Core API like "follow activity log" and describe a high-level scenario that we want to support, and it should fail when the setup is not called (i.e. without the changes you introduced here).

Considering our deadlines, we can land this PR with no new tests and add the tests in a follow-up PR, if you prefer.

Reply:
I'm thinking the latter. Perfect!